igor-core/src/main/java/com/netflix/spinnaker/igor/history/model/Event.java

```diff
@@ -0,0 +1,4 @@
+package com.netflix.spinnaker.igor.history.model;
+
+/** TODO(rz): Document. */
```

Comment:
Would probably be good to document these top level interfaces / how they will be used before some of the more specific things.